just.config.js

```diff
@@ -15,8 +15,12 @@ task('react/start', () => step('react-app-rewired start'))
 task('react/build', () => step('react-app-rewired build'))
 task('react/test', () => step('react-app-rewired test'))
 
-task('lint', () => lintCommand('check'))
-task('lint/fix', () => lintCommand('write', 'warn'))
+task('lint', () => prettierCommand('check'))
+task('lint/fix', () => prettierCommand('write', 'warn'))
+task('lint/strictonce', async () => {
+    await step('eslint --ext tsx,ts ./src/ --cache', { withWarn: true })
+    await step('prettier --check "./src/**/*.{ts,tsx}" --loglevel warn ', { withWarn: true })
```

Comment:
I think these two tasks should run in parallel so they can fail at a single CI run